go.mod

```diff
@@ -40,14 +40,12 @@ require (
 	github.com/stretchr/testify v1.5.1
 	github.com/ti-mo/conntrack v0.3.0
 	github.com/vishvananda/netlink v1.1.0
-	github.com/vmware/go-ipfix v0.2.1
-	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
+	github.com/vmware/go-ipfix v0.2.0
```

Comment:
Is the downgrade and above bincover update expected?

Reply:
No. Fixed it. May be an error when rebasing the PR.